Doc/howto/logging.rst

```diff
@@ -182,10 +191,8 @@ following example::
    logging.basicConfig(level=numeric_level, ...)
 
 The call to :func:`basicConfig` should come *before* any calls to
-:func:`debug`, :func:`info`, etc. Otherwise, those functions will call
-:func:`basicConfig` for you with the default options. As it's intended as a
-one-off simple configuration facility, only the first call will actually do
-anything: subsequent calls are effectively no-ops.
+:func:`logger.debug`, :func:`logger.info`, etc. Otherwise, that logging
+will not be handled.
```

Comment:
Not strictly true. For `WARNING` and higher, a handler of last resort outputs the event to `sys.stderr`.

Reply:
I only discovered this behavior recently 😛 (and have mixed feelings on it). I adjusted it to be technically correct without getting into the details of the last resort handler; but if you would prefer I mention it I can do that as well.